### PR TITLE
support external endpoint to pod/node and loadbalancer ip

### PR DESCRIPTION
**Pull request by Lyt99**

### CODE REVIEW

File: pkg/skoop/infra/aliyun/cloud_manager.go [hunk 1]
```diff
@@ -15,6 +16,7 @@ import (
 type CloudManager struct {
 	vpcID  string
 	region string
+	cidrs  []*net.IPNet
```
Comment (BSWANG) [inline pkg/skoop/infra/aliyun/cloud_manager.go:19]: named `vpcCIDRs` to avoid ambiguous

File: pkg/skoop/network/aliyun/assertion.go [hunk 2]
```diff
@@ -141,14 +144,47 @@ func (a *vpcAssertion) AssertSecurityGroup(srcECS, dstECS string, pkt *model.Pac
...
+			dstECSSecurityGroup = dstECSInfo.Network.SecurityGroups
+		} else {
+			dstECSSecurityGroup = dstENI.SecurityGroups
+		}
+
+		sgs := lo.Values(dstECSSecurityGroup)
+		if len(sgs) > 0 {
+			sgPolicyVerdict := securityPolicyVerdictDrop
+			if sgs[0].Type == aliyun.SecurityGroupTypeNormal {
+				sgPolicyVerdict = securityPolicyVerdictAccept
```
Comment (BSWANG) [inline pkg/skoop/network/aliyun/assertion.go:172]: for ingress direction, default policy verdict always be drop ever type is normal or enterprise.

File: pkg/skoop/network/aliyun/infrashim.go [hunk 3]
```diff
@@ -108,7 +181,52 @@ func (s *aliyunInfraShim) getECSFromNode(node *v1.Node) (string, error) {
 	return providerIDs[1], nil
 }
 
+func (s *aliyunInfraShim) getCLBIDFromService(svc *v1.Service) (*slb.DescribeLoadBalancersResponseBodyLoadBalancersLoadBalancer, error) {
+	if id, ok := svc.Labels["service.k8s.alibaba/loadbalancer-id"]; ok {
```
Comment (BSWANG) [inline pkg/skoop/network/aliyun/infrashim.go:185]: user specify SLB may not have loadbalancer-id field?
Get SLB by ip address?

**Review by BSWANG — CHANGES_REQUESTED** [review]

File: pkg/skoop/infra/aliyun/cloud_manager.go (hunk 1 above)
Comment (Lyt99) [reply pkg/skoop/infra/aliyun/cloud_manager.go]: Thanks! Renamed.

File: pkg/skoop/network/aliyun/assertion.go (hunk 2 above)
Comment (Lyt99) [reply pkg/skoop/network/aliyun/assertion.go]: Fixed.

**Review by BSWANG — APPROVED** [review]

**Pull request merged**